### PR TITLE
Account page

### PR DESCRIPTION
**Pull request by AnnelieseGerrard**

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -4,4 +4,11 @@ def home
 
   def thankyou
   end
+
+  def account
+    @bookings = current_user.bookings
+    @items = current_user.items
+    raise
```
Comment (zyehia) [inline app/controllers/pages_controller.rb]: Can you remove this `raise`?

**Review by zyehia — CHANGES_REQUESTED** [review]
just a tiny edit required.

**Comment by zyehia** [comment]
@AnnelieseGerrard has advised that there are stylistic issues with this commit. I have agreed to work on them.

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (gcarter89) [reply app/controllers/pages_controller.rb]: This has been removed.

**Comment by AnnelieseGerrard** [comment]
> @AnnelieseGerrard has advised that there are stylistic issues with this commit. I have agreed to work on them.

@zyehia sure or i can work on them tonight/tomorrow. at the moment I wanted to get everything working first in this branch and will create another branch later for style amends.

**Review by zyehia — APPROVED** [review]

**Pull request merged**